qtdialogs/DlgSetupManager.py

```diff
@@ -0,0 +1,1197 @@
+            if isLegacy:
+               stateText = 'Legacy'
+            elif loadStateStr.endswith('.encrypted'):
+               stateText = 'Encrypted (locked)'
+            else:
+               stateText = 'Ready'
+            item.setText(2, stateText)
+
+            # Action widget
+            actionWidget = QtWidgets.QWidget()
+            actionLayout = QtWidgets.QHBoxLayout(actionWidget)
+            actionLayout.setContentsMargins(0, 0, 0, 0)
+            actionBtn = QtWidgets.QPushButton()
+            actionBtn.setStyleSheet(UI_STYLE_BUTTON_STANDARD)
+
+            fullPath = walletPath if walletPath else os.path.join(self.armoryDataDirEdit.text(), fileName)
+
+            if isLegacy:
+               actionBtn.setText(self.tr('Migrate'))
+               actionBtn.clicked.connect(lambda _, p=fullPath: self._migrateWallet(p))
+            elif loadStateStr.endswith('.encrypted'):
+               actionBtn.setText(self.tr('Decrypt'))
+               actionBtn.clicked.connect(lambda _, wid=walletId: self._unlockWallet(wid))
+            else:
+               actionBtn.setText(self.tr('Load'))
+               actionBtn.clicked.connect(lambda _, wid=walletId: self._loadWallet(wid))
+
+            actionLayout.addWidget(actionBtn)
+            self.walletList.addTopLevelItem(item)
+            self.walletList.setItemWidget(item, 3, actionWidget)
+
+      except Exception as e:
+         LOGEXCEPT('Error loading wallet list: %s', str(e))
+
+   def _migrateWallet(self, walletPath):
+      try:
+         # DlgMigrateWallet(promptId, wltID, verbose, parent=None, main=None)
+         DlgMigrateWallet(None, walletPath, self.tr('Migrate'), self, self).exec_()
+         TheBridge.wltManager.loadWallets()
+         self.loadWalletList()
+      except Exception as e:
+         QtWidgets.QMessageBox.warning(self, self.tr('Migration Failed'), str(e))
+
+   def _unlockWallet(self, walletId):
+      try:
+         # Control header unlock requires a path; list again to find the entry
+         wltList = TheBridge.wltManager.listWallets()
+         pathForId = None
+         for key in wltList:
+            entry = wltList[key]
+            if getattr(entry, 'walletId', '') == walletId:
+               pathForId = key
+               break
+         if not pathForId:
+            QtWidgets.QMessageBox.warning(self, self.tr('Unlock Failed'), self.tr('Wallet not found.'))
+            return
+         # Use UnlockWalletHandler which integrates with bridge prompts
+         dlg = UnlockWalletHandler(walletId, self.tr('Unlock Wallet'), self)
+         dlg.exec_()
+         TheBridge.wltManager.loadWallets()
+         self.loadWalletList()
+      except Exception as e:
+         QtWidgets.QMessageBox.warning(self, self.tr('Unlock Failed'), str(e))
+
+   def _loadWallet(self, walletId):
+      try:
+         # Stage and refresh
+         TheBridge.wltManager.stageWallet(walletId, True)
```

Comment:
why is this called `_loadWallet` and all it does is staging a wallet? Why are you called `loadWallets` after flipping a single boolean? this stuff is meant to run on more than a rtx5090